src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.CinemaHall;
+import mate.academy.service.CinemaHallService;
+
+@Service
+public class CinemaHallServiceImpl implements CinemaHallService {
+    @Inject
+    private CinemaHallDao cinemaHallDao;
+
+    @Override
+    public CinemaHall add(CinemaHall cinemaHall) {
+        return cinemaHallDao.add(cinemaHall);
+    }
+
+    @Override
+    public CinemaHall get(Long id) {
+        Optional<CinemaHall> cinemaHallOptional = cinemaHallDao.get(id);
+        if (cinemaHallOptional.isEmpty()) {
+            throw new DataProcessingException("CinemaHall with ID: " + id + " not found");
+        }
+        return cinemaHallOptional.get();
```

Comment:
your variant is ok, also we can do more compact with inline `.orElseThrow(() -> new RuntimeException("..."));`